Connectors/Rhino/Speckle.Connectors.RhinoShared/Registration/ServiceRegistration.cs

```diff
@@ -34,10 +34,10 @@ public static void AddRhino(this IServiceCollection serviceCollection, bool isCo
       serviceCollection.AddSingleton<PlugIn>(SpeckleConnectorsRhinoPlugin.Instance);
       serviceCollection.AddSingleton<Command>(SpeckleConnectorsRhinoCommand.Instance);
       serviceCollection.AddDUI<DefaultThreadContext, RhinoDocumentStore>();
+      serviceCollection.AddDUIView();
```

Comment:
No need for DUI for the importer

Reply:
I'm only registering DUIView here if `isConnector` is true. Instead of it being registered always.
<img width="3601" height="1631" alt="image" src="https://github.com/user-attachments/assets/ae9515fc-ccf1-4a78-9f08-a35a355d70ac" />
so, no change for connectors.